tests/target/configs/version/mapped.rs

```diff
@@ -0,0 +1,9 @@
+// rustfmt-version: Two
```

Comment:
Just a small test capturing that the the legacy `version` option is still supported